No scraping error text when transient is not existing

Georg-Git opened the pull request:
No scraping error text when transient is not existing

Trac ticket: 
[https://core.trac.wordpress.org/ticket/62105](https://core.trac.wordpress.org/ticket/62105)

Georg-Git committed:
No scraping error text when transient is not existing

No scraping error text when transient is not existing.
see:
https://core.trac.wordpress.org/ticket/62105

Debarghya-Banerjee commented on src/wp-includes/load.php:
```diff
@@ -1807,6 +1807,10 @@ function wp_start_scraping_edited_file_errors() {
 	$key   = substr( sanitize_key( wp_unslash( $_REQUEST['wp_scrape_key'] ) ), 0, 32 );
 	$nonce = wp_unslash( $_REQUEST['wp_scrape_nonce'] );
 
+	if ( get_transient( 'scrape_key_' . $key ) === false ) {
```
```suggestion
	if ( false === get_transient( 'scrape_key_' . $key ) ) {
```

Add Yoda Condition check.

Debarghya-Banerjee reviewed (changes requested):
Left a suggestion.

Georg-Git committed:
No scraping error text when transient is not existing v2

No scraping error text when transient is not existing

Trac ticket:
https://core.trac.wordpress.org/ticket/62105
